.github/workflows/build.yml

```diff
@@ -51,6 +55,19 @@ jobs:
               - 'libs/content-watcher-lib/**'
               - 'libs/storage/**'
 
+  test-start-sh:
+    name: 'Bats start.sh Tests'
+    runs-on: ubuntu-latest
+    needs: service-matrix
+    strategy:
+      fail-fast: false
+    steps:
+      - name: 'Bats start.sh Tests'
+        if: ${{ needs.service-matrix.outputs.run-all == 'true' }}
+        run: |
+          npm run test:bats-install
+          npm run test:bats-start
+
```

Comment:
Now that this is its own job, you need to check out the repo, install dependencies, etc, which is why the job is currently failing.